model/pail_storage_test.go

```diff
@@ -1,104 +0,0 @@
-package model
-
-import (
-	"context"
-	"fmt"
-	"io/ioutil"
-	"net/http"
-	"os"
-	"strconv"
-	"strings"
-	"testing"
-
-	"github.com/evergreen-ci/pail"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
-
-func TestGetDownloadURL(t *testing.T) {
-	if skip, _ := strconv.ParseBool(os.Getenv("SKIP_INTEGRATION_TESTS")); skip {
```

Comment:
Should pail_storage.go also be deleted?

Reply:
It's still in use by the buildlogger stuff, but I'm not sure if that's in use at all? 

The reason I just deleted this test is because it was storing performance results in pail/s3 and it didn't seem worth it to try to recreate the test for the buildlogger system, when I wasn't even sure that was being used or actively developed on.